templates/firefox-ios.xliff

```diff
@@ -3406,6 +3554,25 @@ Private Tab</source>
       </trans-unit>
     </body>
   </file>
+  <file original="Shared/Supporting Files/en.lproj/LoginsHelper.strings" datatype="plaintext" source-language="en" target-language="en">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="15.0.1" build-num="15A507"/>
+    </header>
+    <body>
+      <trans-unit id="LoginsHelper.DontSave.Button.v122" xml:space="preserve">
+        <source>Now Now</source>
```

Comment:
@adudenamedruby
Pretty sure this and the following string are supposed to say "Not Now".

![rPov1nE](https://github.com/mozilla-l10n/firefoxios-l10n/assets/3644868/6b1a113f-2e69-4ce1-9285-ce569a7f9ae1)

Reply:
LOL! Good catch!